src/hiveboard_bridge/src/TCPServer.cpp

```diff
@@ -0,0 +1,73 @@
+#include "../include/hiveboard_bridge/TCPServer.h"
```

Comment:
Change to #include hiveboard_bridge/TCPServer.h

Reply:
Good catch. Automatic refactoring in CLion seems to have changed the paths of the included files.